frontend/src/pages/Player/styles.css.ts

```diff
@@ -68,6 +68,7 @@ export const playerCenterColumn = style({
 	flexGrow: 1,
 	position: 'relative',
 	minWidth: MIN_CENTER_COLUMN_WIDTH,
+	zIndex: 0,
```

Comment:
Was causing some layout issues with player elements appearing on top of the network request panel.